### PR TITLE
Emit an incoming stream if the onchannel callback is ommited

### PR DESCRIPTION
**Pull request by diasdavid**
I've added this small feature to multiplex that is convenient to the libp2p wrapper for it.

### CODE REVIEW

**Comment by mafintosh** [comment]
i'd be ok with just always emitting 'stream' as well as calling the onchannel function (if specified). i think that's simpler to understand

File: test.js [hunk 1]
```diff
@@ -260,3 +260,26 @@ test('quick message', function (t) {
...
+      stream.end()
+    })
+
+    var stream = plex1.createStream()
+    stream.on('data', function (data) {
+      t.same(data, new Buffer('hello world'))
+      stream.end()
+      t.end()
+    })
+  }, 100)
```
Comment (mafintosh) [inline test.js]: was the setTimeout needed to get this working?

**Comment by diasdavid** [comment]
sounds good to me :) amended my commit to always emit the 'stream' event, with onStream callback or not.

File: test.js (hunk 1 above)
Comment (diasdavid) [inline test.js]: It wasn't, just removed that as well. Thank you for catching that!

**Comment by diasdavid** [comment]
sweet, thanks :D

**Comment by mafintosh** [comment]
out in 6.7.0

**Pull request merged**